Subscriber/CheckoutSubscriber.php

```diff
@@ -381,7 +381,7 @@ private function addAdyenGooglePay(Shopware_Controllers_Frontend_Checkout $subje
                 'merchantName' => Shopware()->Shop()->getName()
             ],
         ];
-        if ($this->configuration->getEnvironment() === Configuration::ENV_LIVE) {
+        if ($this->configuration->getEnvironment() === strtolower(Configuration::ENV_LIVE)) {
```

Comment:
Hi @isfett , thanks for spotting this and thanks for the PR.
Since the [`configuration->getEnvironment`](https://github.com/Adyen/adyen-shopware5/blob/cb03b94a3c9a57d705e2239c6a508369defe2843/Components/Configuration.php#L49) function already allows you to specify the case, why don't you just use that?
```suggestion
        if ($this->configuration->getEnvironment(null, true) === Configuration::ENV_LIVE) {
```